system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/common/Table.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import MUIDataTable from "mui-datatables";
+import PropTypes from "prop-types";
+import React from "react";
+import {withStyles} from "@material-ui/core/styles";
+
+const styles = (theme) => ({
+    table: {
+        minWidth: 1020
+    },
+    tableWrapper: {
+        overflowX: "auto",
+        marginLeft: theme.spacing.unit * 3,
+        marginRight: theme.spacing.unit * 3
+    }
+});
+
+class List extends React.Component {
+
+    render() {
+        const {classes} = this.props;
+        const options = {
+            download: false,
+            selectableRows: false,
+            print: false
+        };
+        const columns = [
+            "Name", "Namespace", "Error Rate", "Average Response Time (s)", "Average Request Count (requests/s)"
+        ];
+        const data = [
+            [" ", " ", " ", " ", " "]
```

Comment:
this will create an empty row which isn't required . please comment

Reply:
this was added to get the UI skeleton, its updated with the real data in development repo https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/common/DataTable.js